src/js/controllers/tab-send.controller.js

```diff
@@ -1,12 +1,38 @@
 'use strict';
 
-angular.module('copayApp.controllers').controller('tabSendController', function(bitcoinUriService, externalLinkService, $scope, $log, $timeout, $ionicScrollDelegate, addressbookService, profileService, lodash, $state, walletService, platformInfo, sendFlowService, gettextCatalog, configService, $ionicPopup, $ionicNavBarDelegate, clipboardService, incomingDataService) {
+angular.module('copayApp.controllers').controller('tabSendController', function tabSendController(
+    bitcoinUriService
+  , externalLinkService
+  , $scope
+  , $log
+  , $timeout
+  , $ionicScrollDelegate
+  , addressbookService
+  , profileService
+  , lodash
+  , $state
+  , walletService
+  , platformInfo
+  , sendFlowService
+  , gettextCatalog
+  , configService
+  , $ionicPopup
+  , $ionicNavBarDelegate
+  , clipboardService
+  , incomingDataService
+  , moonPayService
+) {
   var clipboardHasAddress = false;
   var clipboardHasContent = false;
   var originalList;
+  var isBuyBitcoinAllowed = false;
```

Comment:
It think `buyBitcoinIsAllowed` makes a bit more sense in English. It still contains the useful "is".